src/CheckoutSdk/Sessions/Completion/HostedCompletionInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace Checkout.Sessions.Completion
+{
+    public class HostedCompletionInfo : CompletionInfo
+    {
+        [JsonProperty(PropertyName = "callback_url")]
+        public string CallbackUrl { get; set; }
+
+        [JsonProperty(PropertyName = "success_url")]
+        public string SuccessUrl { get; set; }
+
+        [JsonProperty(PropertyName = "failure_url")]
+        public string FailureUrl { get; set; }
+
+        public HostedCompletionInfo(string callbackUrl, string successUrl, string failureUrl) : base(CompletionInfoType.Hosted)
```

Comment:
Same question about constructors.

Reply:
I based my code on java project. Now I can't remove it because I'm using this contructor.

Reply:
That is not a valid reason for keeping a constructor, also you can change your object instantiation.
Please remove this and all the constructors that are not needed.